kalagraphs.module

```diff
@@ -57,20 +61,31 @@ const KALAGRAPHS_TYPES = [
   'card__program' => [
     'title' => 'Program Card',
     'template' => 'card--program',
-    'path' => '@kalagraphs/molecules',
+    'path' => '@molecules',
     'fields' => [
       'title',
       'text',
       'image',
       'links',
     ],
+  ],
+  'layout__4_8' => [
+    'title' => 'Layout: Sidebar w/Main',
+    'template' => 'layout--4-8',
+    'path' => '@layouts',
+    'fields' => [
+      'subcomponents',
+    ],
     'bundles' => ['kalagraphs_component'],
```

Comment:
Do we need `'bundles' => ['kalagraphs_component']`. I see that you have removed this line from other components.